charts/gardener-extension-provider-alicloud/templates/configmap.yaml

```diff
@@ -25,7 +25,9 @@ data:
 {{- if .Values.config.kubeAPIServer }}
     kubeAPIServer:
       mutateExternalTrafficPolicy: {{ .Values.config.kubeAPIServer.mutateExternalTrafficPolicy }}
-{{- end}}
+{{- end }}
+    service:
+      backendLoadBalancerType: {{ .Values.config.service.backendLoadBalancerType }}
```

Comment:
Please rename backendLoadBalancerType to `backendLoadBalancerSpec` https://help.aliyun.com/document_detail/27577.html?spm=a2c4g.11174283.6.696.1c021192tmG2eY